azureiot.sln

```diff
@@ -62,9 +62,77 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Provisioning.Client.UnitTests", "provisioning\device\tests\Microsoft.Azure.Devices.Provisioning.Client.UnitTests.csproj", "{8797C176-398A-4347-B8E9-38FAF5746EC4}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{4C9BAEB7-5506-4840-A67D-704CFF714A33}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{0F7F505C-CB9A-4CFD-855C-BDB0A9067AD1}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DeviceReconnectionSample", "iothub\device\samples\DeviceReconnectionSample\DeviceReconnectionSample.csproj", "{EB2ADB57-87D4-4AF2-A8F8-4FB48AD56E9D}"
```

Comment:
This is the early sample I brought over, which will now be replaced by the new one (along with all the others). So I'm removing it.